cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTestUtils.java

```diff
@@ -42,16 +42,23 @@ private AnomalyDetectorTestUtils() {
   /**
    * Create history load for broker.
    * @param metricValueById A set of metrics with corresponding load value.
+   * @param metricVarianceById A set of metrics with corresponding load variance.
    * @param numWindows  Number of windows to populate.
    * @param broker The subject broker.
    * @return The load for the broker.
    */
-  public static Map<BrokerEntity, ValuesAndExtrapolations> createHistory(Map<Short, Double> metricValueById, int numWindows, BrokerEntity broker) {
+  public static Map<BrokerEntity, ValuesAndExtrapolations> createHistory(Map<Short, Double> metricValueById,
+                                                                         Map<Short, Double> metricVarianceById,
```

Comment:
The term `variance` refers to the square of the standard deviation.
Does this variable refer to variance or something else?

Reply:
Here it refers to maximal fluctuation.